floor/floor/controller/controller.py

```diff
@@ -109,17 +109,17 @@ def handle_input_event(self, event_name, num, value):
             logger.warning('Ignoring unknown event {}'.format(event_name))
 
     def square_weight_on(self, index):
-        if index > 63 or index < 0:
+        if index > 64 or index < 1:
```

Comment:
I could have updated the MIDI mapping file, but it makes more sense to me to 1-index the tiles.

Reply:
cautiously accepting & remembering this change, should i ever get caught wasting an afternoon related to different 0-index'd & 1-indexed realms... :)

Reply:
Heh, well, I figured that these are really labels, not indexes.